lib/HapiSpool.ts

```diff
@@ -45,12 +45,13 @@ export class HapiSpool extends ServerSpool {
   configure () {
     this.app.config.set('web.server', 'hapi')
     this.app.config.set('web.routes.files.relativeTo', this.app.config.get('main.paths.root'))
+    this.serverConfig = this.app.config.get('web.options');
 
-    this.serverConfig = {
+    _.defaultsDeep(this.serverConfig, {
```

Comment:
We should probably just import defaultsDeep from lodash as `import  {defaultsDeep} from 'lodash'`